doc/esmvalcore/preprocessor.rst

```diff
@@ -54,7 +88,7 @@ it and to provide the corresponding CMOR table. This is to guarantee the proper
 metadata definition is attached to the derived data. Such custom CMOR tables
 are collected as part of the `ESMValTool core package
 <https://github.com/ESMValGroup/ESMValCore>`_. By default, the variable
-derivation will be applied only if not already available in the input data, but 
+derivation will be applied only if not already available in the input data, but
```

Comment:
This sentence is slightly confusing. Maybe rephrase to "By default, the variable derivation will be applied only if the variable is not already available in the input data, ..."

Reply:
Done.